x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/connector/ConnectorIndexServiceTests.java

```diff
@@ -65,6 +67,29 @@ public void testDeleteConnector() throws Exception {
         expectThrows(ResourceNotFoundException.class, () -> awaitDeleteConnector(connectorIdToDelete));
     }
 
+    public void testUpdateConnectorConfiguration() throws Exception {
+        Connector connector = ConnectorTestUtils.getRandomConnector();
+
+        DocWriteResponse resp = awaitPutConnector(connector);
+        assertThat(resp.status(), anyOf(equalTo(RestStatus.CREATED), equalTo(RestStatus.OK)));
+
```

Comment:
You could add an assertion on the initial status to make sure that they are not the same on coincidence.